Loop/Managers/LoopAppManager.swift

```diff
@@ -533,7 +533,8 @@ extension LoopAppManager: UNUserNotificationCenterDelegate {
                                                                          doubleValue: carbAmount),
                                                     startDate: mealTime,
                                                     foodType: nil,
-                                                    absorptionTime: nil)
+                                                    absorptionTime: nil,
+                                                    absorptionData: nil)
```

Comment:
should be nil here because this is for missed meal detection (creating from scratch).